platforms/common/include/px4_platform_common/i2c_spi_buses.h

```diff
@@ -236,11 +236,11 @@ class I2CSPIDriver : public I2CSPIDriverBase
 
 	virtual ~I2CSPIDriver() = default;
 
-	void Run() final {
+	void Run() final
+	{
```

Comment:
Can you revert these lines? It's an astyle version problem.

Reply:
I should have disabled the pre-commit astyle hook check which prevented me reverting instead of manually updating